Listener/SoftLockListener.php

```diff
@@ -17,7 +17,6 @@ public function __construct($maintenancePage, $maintenanceLock, array $whitePath
     {
         $this->maintenancePage = $maintenancePage;
         $this->lock = file_exists($maintenanceLock);
-        $this->whiteIps = $this->whiteIps;
```

Comment:
Why this assignment is pointless? The `whiteIps` assignement comes from the services list